platform/xbox/functions/_PDCLIB/_PDCLIB_open.c

```diff
@@ -13,9 +13,12 @@
 
 #include "pdclib/_PDCLIB_glue.h"
 
-#include <hal/fileio.h>
+#include <errno.h>
+#include <windows.h>
 
-int _PDCLIB_open( const char * const filename, unsigned int mode )
+int _PDCLIB_w32errno( DWORD werror );
+
+void *_PDCLIB_open( const char * const filename, unsigned int mode )
```

Comment:
nit: Why doesn't this return `_PDCLIB_fd_t`? What does upstream pdclib do here in its reference implementation?

Reply:
PDCLib's example port uses `int` here (which is the same as its `_PDCLIB_fd_t`.

Reply:
CC @DevSolar. This feels like a bug.
I think we should replace it by `_PDCLIB_fd_t`?

Reply:
It is indeed a bug, which got fixed in SVN @856 / GIT 7a65248 (2019-12-19).

Reply:
In that case it's no longer a nit for me; please fix @thrimbor

Reply:
Fixed.